pyflakes/test/test_other.py

```diff
@@ -161,6 +161,15 @@ def _(): pass
         def _(): pass
         ''')
 
+    def test_redefinedUnderscoreImportation(self):
+        """
+        Test that shadowing an underscore importation raises a warning.
+        """
+        self.flakes('''
+        from .i18n import _
+        def _(): pass
+        ''', m.RedefinedWhileUnused)
+
```

Comment:
Can you also add a test that doing

```py
from . import i18n

def i18n(): pass
```

Raises a `RedefinedWhileUnused` warning?